spec/javascript/e2e/SupplementalApplicationPage.e2e.js

```diff
@@ -21,7 +21,7 @@ describe('SupplementalApplicationPage', () => {
 
```

Comment:
at some point soon we may want to think about breaking this up into multiple files so it doesn't get too long. For now, do you think it might help to add another layer of "describe" to help organize the e2e tests a bit?

Reply:
yeah i've also been thinking that - i wonder if there's any problem with doing that breaking up now? @fddayan @chrisdolendo how would you guys feel about breaking up the supp app e2e file now?

Reply:
👍I'm ok with splitting the file if it's growing. We could re think the naming convention we use for e2e tests. Instead of using the Page as the main test we can can `SupplementalApplicationPage/feature_name.e2e.test` 
like 
`SupplementalApplicationPage/manage_rental_asistances.e2e.test` .... or any name

Reply:
@akegan Fed and I have broken apart the supp app e2e tests in a couple of recent commits - please review.